### PR TITLE
Feat/sqlite

### PR DESCRIPTION
**Pull request by Yzorek**
## Requirements

- [x] I have read the `CODE_OF_CONDUCT.md`.
- [x] I have updated `CHANGELOG.md` & `tools/install/deb/changelog` to reflect my changes.

### CODE REVIEW

File: src/plugins/vsmtp-plugin-sqlite/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE.  See the GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License along with
+ * this program. If not, see https://www.gnu.org/licenses/.
+ *
+*/
+
```
Comment (Mathieu-Lala) [inline src/plugins/vsmtp-plugin-sqlite/src/lib.rs:17]: Add the clippy lints

File: src/plugins/vsmtp-plugin-sqlite/Cargo.toml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+crate-type = ["cdylib"]
+
+[dependencies]
+vsmtp-plugin-vsl = { version = "=2.2.1", path = "../../vsmtp/vsmtp-plugin-vsl" }
+rhai = { version = "1.12" }
+r2d2 = { version = "0.8.10", default-features = false }
+rusqlite = { version = "0.29.0", features = ["bundled"] }
+serde = { version = "1.0.152", default-features = false, features = ["std", "derive"] }
+humantime-serde = { version = "1.1.1", default-features = false }
+smartstring = { version = "*", features = ["proptest", "serde"] }
```
Comment (Mathieu-Lala) [inline src/plugins/vsmtp-plugin-sqlite/Cargo.toml:32]: Set a full version for `smartstring` and why is there `proptest` ?

Comment (Mathieu-Lala) [inline src/plugins/vsmtp-plugin-sqlite/Cargo.toml:32]: Set a full version for `smartstring` and why is there `proptest` ?

**Review by Mathieu-Lala — CHANGES_REQUESTED** [review]

File: src/plugins/vsmtp-plugin-sqlite/src/lib.rs (hunk 1 above)
Comment (ltabis) [reply src/plugins/vsmtp-plugin-sqlite/src/lib.rs:17]: which one ?

Comment (Mathieu-Lala) [reply src/plugins/vsmtp-plugin-sqlite/src/lib.rs:17]: All of them :)

**Comment by ltabis** [comment]
Closing this since we release vSMTP v3.

**Pull request closed without merging**